InterfacePython/testAddons.py

```diff
@@ -136,8 +136,43 @@ def checkUnstructuredL2(self):
 
         self.assertLess(np.max(np.abs(model - surrogate[:,0])), 5.E-3, "Constructed from unstructured too inaccurate.")
 
+    def checkExoticQuadrature(self):
+        # Initialize some useful constants and utility functions.
+        integrand = lambda x : np.exp(-np.linalg.norm(x) ** 2)
+        sinc1s0 = lambda x : np.sinc(x / np.pi)
+        sinc10s1 = lambda x : np.sinc(10 * (x - 1) / np.pi)
+        max_level = 25
+        nref = 101
+        def create_surrogate(lambda_fn):
+            grid = Tasmanian.makeGlobalGrid(1, 1, nref, "level", "gauss-legendre")
+            grid.loadNeededValues(np.array([lambda_fn(x) for x in grid.getNeededPoints()]))
+            return grid
+        def compute_integral(grid, integrand):
+            return np.sum([integrand(x) for x in grid.getPoints()] * grid.getQuadratureWeights())
+        # Each pair below consists of a 1D integral value (first) and a CustomTabulated object (second) used to approximate this integral.
+        testPairs = [
+            [1.4321357541271255, Tasmanian.createExoticQuadratureFromFunction(max_level, 0.0, sinc1s0, nref, "Sinc1s0-shift0-Symm-byFn", True)],
```

Comment:
Use scientific notation, which will align the first column of numbers and make it cleaner visually, e.g.,
```
[1.4321357541271255E-00, Tasmanian ...
[6.4062055930705356E-02, Tasmanian ...
```

Reply:
Done.